web-app/src/app/tabs/controls/controls.component.css

```diff
@@ -0,0 +1,161 @@
+.header {
```

Comment:
This is currently overriding the one in `styles.css`, if that's not intended, you'll need to rename it. If it is, group them all together under a comment